doc/installing.rst

```diff
@@ -38,36 +38,153 @@ System Requirements
 .. note::
    For Linux distributions, we support, if technically feasible, the latest 2 versions per platform and the previous Ubuntu `LTS`_.
 
+Customizing the Windows installation
+----------------------------------
+
+If you just want to install ownCloud Desktop Synchronization Client on your local
+system, you can simply launch the .msi file and configure it in the wizard
+that pops up.
+
+Features
+^^^^^^^^
+
+The MSI installer provides several features that can be installed or removed
+individually, which you can also control via command-line, if you are automating
+the installation::
+
+   msiexec /passive /i ownCloud-x.y.z.msi
+
+will install the ownCloud Desktop Synchronization Client into the default location
+with the default features enabled. If you want to disable e.g. desktop shortcut
```

Comment:
Small nit-pick, pls change "e.g." to "e.g.,".

Reply:
According to this https://jakubmarian.com/comma-after-i-e-and-e-g/ I should probably make it `, e.g., ` right?